content/v0.80.0.md

```diff
@@ -0,0 +1,478 @@
+<!-- /vscode-markdown-toc -->
+
+
+##  2. <a name='Summary'></a>Summary
+
+ReelMagic DOS games are now playable thanks to the efforts of Jon Dennis, Chris
+Guthrie, and Joseph Whittaker, with testing and feedback from Vogons users Uka,
+DonutKing, and Vetz. Mouse emulation has been improved and now supports up to
+seven independent mice, enabling split-screen play in games like Settlers 1 and
+Settlers II. More settings, including video, audio, and MIXER effects, can now
+be changed on the fly. A new `--working-dir` command line flag allows DOSBox
+Staging to launch from a specific working directory. Repo packagers should note
+that `SDL2-image` is now a dependency.
+ 
+Read on for all the details!
+
+##  3. <a name='Howtoupgrade'></a>How to upgrade
+
+Get started by downloading the latest version and updating your configuration
+file:
+
+**Downloads:**
+
+* [Linux](/downloads/linux/)
+* [Windows](/downloads/windows/)
+* [macOS](/downloads/macos/)
+
+**Upgrade your configuration file:**
+
+1. Launch the new release.
+2. At the`Z:\>` prompt, run: `config -wc` to write the new configuration file. 
+
+Notes:
+
+- Existing settings will be retained, but extraneous comments will be dropped.
+
+- Starting with version 0.80, the configuration file is written in the lauguage
+  in which DOSBox Staging was started.
+- To write an English configuration file, launch with the `-lang en`
+  command-line flag (to start DOSBox Staging in English) and then (re-)write
+  your configuration file.
+
+
+##  4. <a name='Newfeatures'></a>New features
+
+
+###  4.1. <a name='ReelMagicsupport'></a>ReelMagic support
+
+DOS ReelMagic games are now playable thanks to Jon Dennis's
+(patch)[https://github.com/jrdennisoss/dosboxrm] with help from Chris Guthrie
+and Joseph Whittaker.
+
+See the `[reelmagic]` section in your configuration file to enable this. We
+suggest only enabling ReelMagic emulation on an as-needed basis.
+
+<video controls width=640>
+  <source src="/reelmagic_rtz.mp4" type="video/mp4">
+  Your browser does not support the <code>video</code> element.
+</video>
+
+Jon plans to add DMA support (primarily used by the ReelMagic Win 3.1 driver and
+games) when time permits, but most ReelMagic games for DOS are already fully
+functional and playable.
+
+###  4.2. <a name='Mouseimprovements'></a>Mouse improvements
+
+**Dedicated configuration section:**
+
+The mouse configuration has been moved to its own section and additional options
+have been added. A tool called `mousectl` is available on `Z:\>` for more
+detailed per-interface mouse configuration."
+ 
+**Dual mouse gaming:**
+
+Multiple physical mice can now be mapped to specific emulated mouse interfaces,
+enabling two-player split-screen gaming (The Settlers, The Settlers II) using
+two USB mice on a single computer.
+
+<video controls width=640>
+  <source src="/dual_mouse_s2.mp4" type="video/mp4">
+  Your browser does not support the <code>video</code> element.
+</video>
```

Comment:
Please add the video in `webm` format as well, as per my below comment.

Reply:
(the h.264-encoded mp4's should be everything-compatible)